bitnami/contour/values-production.yaml

```diff
@@ -257,6 +257,27 @@ envoy:
   ##
   podAnnotations: {}
 
+  ## Pod security context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ##
+  podSecurityContext: {}
+
+  ## Envoy container security context - envoy needs to run as root to bind to 80, 443
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ##
+  containerSecurityContext:
+    runAsUser: 0
```

Comment:
I agree with adding podSecurityContext and podSecurityContext parameters. That said, they should follow the structure:

```yaml
podSecurityContext:
  enabled: true or false
  (...)
containerSecurityContext:
  enabled: true or false
  (...)
```

Check this example: https://github.com/bitnami/charts/blob/master/bitnami/mongodb/values.yaml#L261

The reason, with you current values, you cannot remove the securityContext. Try this:

```console
$ cat > values.yaml << 'EOF'
envoy:
  containerSecurityContext: {}
EOF
$ helm template -f values.yaml contour bitnami/contour
```

You'll find out that even you set `{}`, it still sets `runAsUser: 0`